src/geo.c

```diff
@@ -525,7 +525,7 @@ void geoaddCommand(client *c) {
  *                               [COUNT count [ANY]] [STORE key|STOREDIST key]
  * GEORADIUSBYMEMBER key member radius unit ... options ...
  * GEOSEARCH key [FROMMEMBER member] [FROMLONLAT long lat] [BYRADIUS radius unit]
- *               [BYBOX width height unit] [WITHCOORD] [WITHDIST] [WITHASH] [COUNT count [ANY]] [ASC|DESC]
+ *               [BYBOX width height unit] [WITHCOORD] [WITHDIST] [WITHHASH] [COUNT count [ANY]] [ASC|DESC]
  * GEOSEARCHSTORE dest_key src_key [FROMMEMBER member] [FROMLONLAT long lat] [BYRADIUS radius unit]
  *               [BYBOX width height unit] [COUNT count [ANY]] [ASC|DESC] [STOREDIST]
```

Comment:
To make my life easier, I was adding [patterns for various prefixes](https://github.com/jsoref/valkey/blob/f95420b90c7c53ff42172d6b80785eee61ed6bba/.github/actions/spelling/patterns.txt#L5-L82). I'm very glad that I didn't add such a pattern for `WITH`, otherwise I would have missed this as it would have seen `ASH` which is a valid word.

If people want to add check-spelling, I'd probably remove those patterns and instead add the items that were covered allow.txt or expect.txt.